File: app/src/main/java/com/example/weatherapp1/Data/Model/Wind.java
package com.example.weatherapp1.Data.Model;

import android.arch.persistence.room.Ignore;
import android.os.Parcel;
import android.os.Parcelable;

public class Wind extends ClassLoader implements Parcelable {
    private String speed;

    public Wind(String speed) {
        this.speed = speed;
    }

    @Ignore
    public Wind() {
    }

    public String getSpeed () {
        return speed;
    }
    public void setSpeed (String speed) {
        this.speed = speed;
    }

    @Override
    public String toString() {
        return "Wind{" +
                "speed='" + speed + '\'' +
                '}';
    }


    @Override
    public int describeContents() {
        return 0;
    }

    @Override
    public void writeToParcel(Parcel dest, int flags) {
        dest.writeString(this.speed);
    }

    protected Wind(Parcel in) {
        this.speed = in.readString();
    }

    public static final Parcelable.Creator<Wind> CREATOR = new Parcelable.Creator<Wind>() {
        @Override
        public Wind createFromParcel(Parcel source) {
            return new Wind(source);
        }

        @Override
        public Wind[] newArray(int size) {
            return new Wind[size];
        }
    };
}

File: app/src/main/java/com/example/weatherapp1/Data/Model/Wind.kt
package com.example.weatherapp1.Data.Model

import android.arch.persistence.room.Ignore
import android.os.Parcel
import android.os.Parcelable

class Wind : ClassLoader, Parcelable {
    var speed: String? = null

    constructor(speed: String) {
        this.speed = speed
    }

    @Ignore
    constructor() {
    }

    override fun toString(): String {
        return "Wind{" +
                "speed='" + speed + '\''.toString() +
                '}'.toString()
    }


    override fun describeContents(): Int {
        return 0
    }

    override fun writeToParcel(dest: Parcel, flags: Int) {
        dest.writeString(this.speed)
    }

    protected constructor(`in`: Parcel) {
        this.speed = `in`.readString()
    }

    companion object CREATOR : Parcelable.Creator<Wind> {
        override fun createFromParcel(parcel: Parcel): Wind {
            return Wind(parcel)
        }

        override fun newArray(size: Int): Array<Wind?> {
            return arrayOfNulls(size)
        }
    }
}
